Services/Database/classes/MDB2/class.ilDB.php

```diff
@@ -2561,11 +2561,21 @@ public function sanitizeMB4StringIfNotSupported($query)
 		return $query;
 	}
 
+
 	/**
 	 * @inheritDoc
 	 */
 	public function doesCollationSupportMB4Strings()
 	{
 		return false;
 	}
-}
\ No newline at end of file
+
+
+	/**
+	 * @inheritDoc
+	 */
+	public function cast($a_field_name, $a_dest_type): string {
```

Comment:
@lbruening just saw you use PHP7 syntax here, could you please revert this to PHP5.x Synthat without return type declaration. thanks in advance!